.github/pull_request_template.md

```diff
@@ -0,0 +1,49 @@
+Fixes #
+
+
+Please sign your PullRequest
```

Comment:
Is it worth putting "Signed-off-by: ..." here instead of "Please sign your PR"?

Reply:
Are we now requiring "Signed-off-by:" in DeepSea? Unless everyone starts signing their commits, this should just be dropped?

Reply:
good point @tserong 

@smithfarm I think this is best practice and we should at least start remind people of it.